src/GovUk.Education.ExploreEducationStatistics.Publisher/Services/PublishingCompletionService.cs

```diff
@@ -88,20 +91,27 @@ await releaseIdsToUpdate
             .Where(p => publicationSlugs.Contains(p.Slug))
             .Select(p => p.Id)
             .ToListAsync();
-        
+
         await directlyRelatedPublicationIds
             .ToAsyncEnumerable()
             .ForEachAwaitAsync(_publicationRepository.UpdateLatestPublishedRelease);
-        
+
+        // Set the published date on any methodologies used by these publications that are now publicly accessible
+        // as a result of releases being published
+        await directlyRelatedPublicationIds
+            .ToAsyncEnumerable()
+            .ForEachAwaitAsync(publicationId =>
+                _methodologyService.SetPublishedDatesByPublication(publicationId));
```

Comment:
I assume not, but we don't want to update existing associated methodlogies' published dates as well if an `UpdatePublishedDate` is true?

Reply:
I assumed that updating release published dates is independent of methodologies and that methodologies continue to get a published date set if they are being published for the first time (either because the publication has it's first release accessible or a release that the methodology depends on is now accessible).

We confirmed at standup on 09/02/23 with @cjrace that this is correct.